californium-core/src/main/java/org/eclipse/californium/core/network/stack/ObserveLayer.java

```diff
@@ -160,45 +164,47 @@ public void receiveEmptyMessage(Exchange exchange, EmptyMessage message) {
 		}
 		super.receiveEmptyMessage(exchange, message);
 	}
-	
+
 	private void prepareSelfReplacement(Exchange exchange, Response response) {
 		response.addMessageObserver(new NotificationController(exchange, response));
 	}
-	
+
 	/**
 	 * Sends the next CON as soon as the former CON is no longer in transit.
 	 */
 	private class NotificationController extends MessageObserverAdapter {
-		
+
 		private Exchange exchange;
 		private Response response;
-		
+
 		public NotificationController(Exchange exchange, Response response) {
 			this.exchange = exchange;
 			this.response = response;
 		}
-		
+
 		@Override
 		public void onAcknowledgement() {
 			synchronized (exchange) {
 				ObserveRelation relation = exchange.getRelation();
 				final Response next = relation.getNextControlNotification();
-				relation.setCurrentControlNotification(next); // next may be null
+				relation.setCurrentControlNotification(next); // next may be
+																// null
```

Comment:
OK, and the maximum line widht for comments :-) 
(in my californium style its 80)